bve/src/parse/panel1_cfg/sections/digital_indicator.rs

```diff
@@ -0,0 +1,31 @@
+use bve_derive::{FromKVPSection, FromKVPValueEnumNumbers};
+use cgmath::{Array, Vector2};
+
+#[derive(Debug, Clone, PartialEq, FromKVPSection)]
+pub struct DigitalIndicatorSection {
+    pub number: String,
+    pub corner: Vector2<f32>,
+    pub size: Vector2<f32>,
+    pub unit: Unit,
+}
+
+impl Default for DigitalIndicatorSection {
+    fn default() -> Self {
+        Self {
+            number: String::default(),
+            corner: Vector2::from_value(0.0),
+            size: Vector2::from_value(0.0),
+            unit: Unit::default(),
+        }
+    }
+}
+
+#[derive(Debug, Clone, PartialEq, FromKVPValueEnumNumbers)]
+pub enum Unit {
+    #[kvp(default, alias = "km/h")]
+    Kph,
+    #[kvp(alias = "mph")]
+    Mph,
+    #[kvp(alias = "m/s")]
+    Ms,
```

Comment:
This looks like milliseconds or miss. Kph and Mph are common abbreviations, but m/s doesn't translate well into Rust tokens. Maybe spell all these out? It's kinda weird to have `M` represent either miles or meters

Reply:
Fixed